composer.json

```diff
@@ -18,10 +18,10 @@
         }
     },
     "require": {
-        "php": ">=5.5.0",
+        "php": ">=8.1",
         "composer/semver": "^3.2",
-        "symfony/console": "^2.8 || ^3 || ^4 || ^5 || ^6",
-        "symfony/filesystem": "^2.5 || ^3 || ^4 || ^5 || ^6"
+        "symfony/console": "^5.4 || ^6.4",
+        "symfony/filesystem": "^5.4 || ^6.4"
```

Comment:
Further down in this file, change:
```
"dev-main": "2.x-dev"
```
To:
```
"dev-main": "3.x-dev"
```
Be sure to never tag the changes in this PR with a 2.x version. Probably fork a 2.x branch off of main before merging this PR.

Reply:
Yep. I'm not familiar with the preferred workflow on this project, but typically I would merge this PR to main, cut a 3.0 release, and then in the future cut a 2.x branch if another 2.x release is required